### DISCUSSION TITLE
How to get native speed working in aio

### DISCUSSION THREAD

**Discussion by jlmanohar** [nextcloud/recognize#647] [Q&A]
Hi, I use Nextcloud-aio which I believe uses alpine image of Nextcloud, how can I get native speed on it ? gcompat seems to be installed which should have provided compatibility for glibc.

**Comment by marcelklehr** [comment]
I don't know the details of Alpine, sadly. If gcompat doesn't work, I'm out of ideas.

**Reply by jlmanohar** [reply]
Thanks, will update incase I find something
